app/models/protocol_subscription.rb

```diff
@@ -34,6 +34,12 @@ def transfer!(transfer_to)
     save!
   end
 
+  def stop_response
+    # We can be sure there is always only one stop response as this is forced / validated in the
```

Comment:
Dit snap ik niet. Er is toch **at most** één stop measurement?

De code die hier staat is wel goed (want die levert in dat geval gewoon nil op), maar het comment is wat verwarrend.